CHANGES.md

```diff
@@ -26,9 +26,6 @@ Unreleased
 - The `interface` and `implementation` fields of a `(dialect)` are now optional
   (#7757, @gpetiot)
 
-- Revert #7415 and #7450 (Resolve `ppx_runtime_libraries` in the target context when
```

Comment:
Since we already released this with the revert, this PR should add a new changes entry rather this entry.